### ISSUE TITLE
Multithreading NER not working

### ISSUE THREAD

**Issue by milsanore** [explosion/spaCy#1910]
## Steps to reproduce
- Create new project
- Create python3 virtual env
- Install spacy via:
  - `pip install -U spacy`
  - `python -m spacy validate`
  - `python -m spacy download en`
- Run sample code
- Monitor CPU usage (I use htop, which reads a load average of 1, and one core working hard)
- Buy a boat

## Environment
* **spaCy version:** 2.0.5
* **Platform:** Linux-3.19.0-25-generic-x86_64-with-Ubuntu-14.04-trusty
* **Models:** en
* **Python version:** 3.4.3

Hi,

Firstly, thanks for the great library. 

I cannot seem to get multithreading to work (related to https://github.com/explosion/spaCy/issues/172 , especially @honnibal 's sample code https://github.com/explosion/spaCy/issues/172#issuecomment-183963403). Here is my code (forgive the quality, I'm a python n00b):

```
import spacy


class MyNer:

    def __init__(self):
        self.nlp = spacy.load('en')

    def evaluate_ner(self, arr_bios):

        arr_bio_tuples = []
        for item_bio in arr_bios:
            arr_bio_tuples.append((
                item_bio['biography'],
                item_bio['id']
            ))

        arr_out = []
        for item_doc in self.nlp.pipe(arr_bio_tuples, as_tuples=True, n_threads=8):
            arr_entities = {}
            for item_entity in item_doc[0].ents:
                if item_entity.label_ not in arr_entities:
                    arr_entities[item_entity.label_] = []
                arr_entities[item_entity.label_].append(item_entity.text)

            ner_output = {
                'user_id': item_doc[1],
                'ner': arr_entities
            }

            arr_out.append(ner_output)

        return arr_out
```

Am I using `nlp.pipe()` incorrectly, or do I need to install spacy in a particular fashion? Any suggestions welcome :)

Thanks

**Comment by milsanore** [comment]
This appears to be happening on my vmware virtual machine, but not on my amazon ec2 instance. So I'll close until I dig up more info. Thanks

**Comment by janecz-n** [comment]
Kind of the same problem for me. Multi thread run on my:

- **Platform:**           Linux-4.13.0-32-generic-x86_64-with-Ubuntu-16.04-xenial
- **Python version:**     3.5.2          
- **spaCy version:**      2.0.5

but not on:

- **Platform:**                Linux-3.14.32-xxxx-grs-ipv6-64-x86_64-with-Debian-8
- **Python version:**     3.4.2          
- **spaCy version:**      2.0.5

So I take any information you could dig :)

**Comment by janecz-n** [comment]
@milsanore, could you provide the `spacy info` for your amazon ec2 instance ?

**Comment by janecz-n** [comment]
Problem solved: It need to be python >= 3.5 for the multi-threading to work by default
